merino-integration-tests/src/utils/test_tools.rs

```diff
@@ -34,6 +34,9 @@ use tracing_subscriber::{fmt::MakeWriter, layer::SubscriberExt};
 ///     ).await
 /// }
 /// ```
+///
+/// # Panics
+/// May panic if tests could not be set up correctly.
```

Comment:
How often do you expect this to happen? Does it make sense to add a troubleshooting section to the documentation for this?